<launch>
	<!-- base_link displacement from the ground -->
	<arg name="base_link_dz" default="0.48" />
	<!-- Publish interval in ms -->
	<arg name="tf_publish_interval" default="10" />

	<node name="vehicle" pkg="kut_ugv_vehicle" type="crio_comm_node" required="true" output="screen">
		<rosparam command="load" file="$(find kut_ugv_vehicle)/config/kia_sorento.yaml" />
		<param name="fixed_frame_id" value="odom" />
		<param name="crio/ip" value="192.168.3.71" />
		<param name="publish_rate" value="30" />
		<param name="publish_odom_tf" value="false" />
		<remap from="motion_command" to="~/motion_command" />
		<remap from="odom" to="~/odom" />
		<remap from="state" to="~/state" />
	</node>

	<!-- ### Coordinate frames ### -->
	<node name="base_link_tf_pub" pkg="tf" type="static_transform_publisher" required="true" output="screen"
		args="0 0 $(arg base_link_dz) 0 0 0 base_footprint base_link $(arg tf_publish_interval)" />
</launch>
